src/modules/endpoint/models/CategoriesContainer.js

```diff
@@ -0,0 +1,72 @@
+import normalizeUrl from 'normalize-url'
```

Comment:
Should we name this CategoriesContainerModel to be more consistent? I don't think it's too long.

Reply:
is it a model?

Reply:
@Schedulaar what do you think?

Reply:
Hmm.. not sure about that, I think it's a container for a model and not a model itself. But our naming is difficult here (since we're in the models directory) :D